doc/040_backup.rst

```diff
@@ -75,8 +75,16 @@ Now is a good time to run ``restic check`` to verify that all data
 is properly stored in the repository. You should run this command regularly
 to make sure the internal structure of the repository is free of errors.
 
-You can exclude folders and files by specifying exclude-patterns. Either
-specify them with multiple ``--exclude``'s or one ``--exclude-file``
+You can exclude folders and files by specifying exclude patterns, currently
+the exclude options are:
+
+-  ``--exclude`` Specified one or more times to exclude one or more items
+-  ``--exclude-caches`` Specified once to exclude folders containing a special file
+-  ``--exclude-file`` Specified one time to exclude items listed in a given file
```

Comment:
This can also be specified multiple times, the lists of files to be excluded are just joined.